docker/bikeshed/README.md

```diff
@@ -0,0 +1,28 @@
+Docker image with `bikeshed`
+===
+
+Build the image:
+
+    alice@work $ docker build -t bikeshed docker/bikeshed --no-cache
```

Comment:
this presumes that one has docker installed on one's machine, yes?  we should say so if so.

Reply:
Yes; ok.